tasks/collector.py

```diff
@@ -80,8 +82,32 @@ def generate(ctx):
                 code=1,
             ) from e
 
-    # Clean the files with main* in comp/otelcol/collector-contrib/impl
+    # Read the output path from the manifest file
     impl_path = "./comp/otelcol/collector-contrib/impl"
+    output_path = None
+    try:
+        with open(config_path) as file:
+            manifest = yaml.safe_load(file)
+            output_path = manifest["dist"]["output_path"]
+    except Exception as e:
```

Comment:
You could probably use a more granular exception here (either `FileNotFoundError` or `yaml.scanner.ScannerError` for instance) to precise what kind of error you expect to happen, wdyt?

Reply:
We are using Exception in rest of the function. I will revisit the pattern later.